pages/productList.html

```diff
@@ -0,0 +1,499 @@
+              <p class="text-secondary-color">50</p>
+              <p class="text-secondary-color">150</p>
+              <p class="text-secondary-color">200</p>
+            </div>
+            <div class="productlist-input-container">
+
+              <input
+                type="range"
+                name="rangeInput"
+                class="slider"
+                min="0"
+                max="100"
+              />
+            </div>
+          </div>
+          <div class="productlist-aside-item">
+            <p class="font-wt-bold">Category</p>
+            <div class="productlist-input-container">
+              <div class="productlist-input-item-hz">
+                <input
+                  type="checkbox"
+                  name="category-checkbox"
+                  id="men-checkbox"
+                  value="men"
+                />
+                <label for="men-checkbox">Men</label>
+              </div>
+              <div class="productlist-input-item-hz">
+                <input
+                  type="checkbox"
+                  name="category-checkbox"
+                  id="women-checkbox"
+                  value="women"
+                />
+                <label for="women-checkbox">Women</label>
+              </div>
+              <div class="productlist-input-item-hz">
+                <input
+                  type="checkbox"
+                  name="category-checkbox"
+                  id="kids-checkbox"
+                  value="kids"
+                />
+                <label for="kids-checkbox">Kids</label>
+              </div>
+            </div>
+          </div>
+          <div class="productlist-aside-item">
+            <p class="font-wt-bold">Rating</p>
+            <div class="productlist-input-container">
+              <div class="productlist-input-item-hz">
+                <input
+                  type="radio"
+                  name="rating-radio"
+                  id="4star-radio"
+                  value="4star"
+                />
+                <label for="4star-radio">4 stars &#38; above</label>
+              </div>
+              <div class="productlist-input-item-hz">
+                <input
+                  type="radio"
+                  name="rating-radio"
+                  id="3star-radio"
+                  value="3star"
+                />
+                <label for="3star-radio">3 stars &#38; above</label>
+              </div>
+              <div class="productlist-input-item-hz">
+                <input
+                  type="radio"
+                  name="rating-radio"
+                  id="2star-radio"
+                  value="2star"
+                />
+                <label for="2star-radio">2 stars &#38; above</label>
+              </div>
+              <div class="productlist-input-item-hz">
+                <input
+                  type="radio"
+                  name="rating-radio"
+                  id="1star-radio"
```

Comment:
I am not sure but I think , id doesn't start with number

Reply:
It is for the label only so I put like that